src/app/lib/presentation/diagnostic/device_info_widget.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:io';
+
+import 'package:app/presentation/diagnostic/diagnostic_text.dart';
+import 'package:flutter/material.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+final class DeviceInfoWidget extends StatelessWidget {
+  const DeviceInfoWidget({super.key});
+
+  Future<List<String>> getBuildAndAppInfo() async {
+    var packageInfo = await PackageInfo.fromPlatform();
+    var data = <String>[
+      '- Platform: ${Platform.operatingSystem}',
```

Comment:
I don't think you need the dash prefixes.

Reply:
This is to answer the other question, I'm not sure why, but I don't see how to reply to it. But basically, router_examples was just a section I added to put pages in it that I added to show richer examples of navigation (there were only two pages)

Reply:
I was thinking maybe it would be best to remove the "base page" with the menu and keep the others while there is still no real navigation?